elasticsearch/connection/pooling.py

```diff
@@ -24,3 +24,10 @@ def _get_connection(self):
 
     def _release_connection(self, con):
         self._free_connections.put(con)
+
+    def close(self):
+        """
+        Explicitly close connection
+        """
+        return True
```

Comment:
is there a reason why there it `True` being returned anywhere? I don't think we need a return value here